### PR TITLE
Added URL's for Paper cited in some files

### PR DESCRIPTION
**Pull request by codeboy5**
While going through a file, I feel I want to reference the paper sometimes to get a better understanding so it is extremely useful to have the links mentioned.

### CODE REVIEW

File: src/mlpack/methods/ann/layer/reparametrization.hpp [hunk 1]
```diff
@@ -39,7 +39,8 @@ namespace ann /** Artificial Neural Network. */ {
  *              Xavier Glorot, Matthew Botvinick, Shakir Mohamed and
  *              Alexander Lerchner | Google DeepMind},
  *   journal = {2017 International Conference on Learning Representations(ICLR)},
- *   year    = {2017}
+ *   year    = {2017},
+ *   url     = {https://openreview.net/references/pdf?id=Sy2fzU9gl}
```
Comment (birm) [inline src/mlpack/methods/ann/layer/reparametrization.hpp]: Is this a stable/permanent url?

Comment (codeboy5) [reply src/mlpack/methods/ann/layer/reparametrization.hpp]: The google deepmind website ( https://deepmind.com/research/publications/beta-VAE-Learning-Basic-Visual-Concepts-with-a-Constrained-Variational-Framework ) leads to this url, so I believe it should be stable .

Comment (zoq) [reply src/mlpack/methods/ann/layer/reparametrization.hpp]: Maybe we should use `https://deepmind.com/research/publications/beta-VAE-Learning-Basic-Visual-Concepts-with-a-Constrained-Variational-Framework` instead?

Comment (codeboy5) [reply src/mlpack/methods/ann/layer/reparametrization.hpp]: Okay @zoq .

File: src/mlpack/methods/ann/layer/alpha_dropout.hpp [hunk 2]
```diff
@@ -36,7 +36,7 @@ namespace ann /** Artificial Neural Network. */ {
  *   title   = {Self-Normalizing Neural Networks},
  *   journal = {Advances in Neural Information Processing Systems},
  *   year    = {2017},
- *   url     = {https://arxiv.org/abs/1706.02515}
+ *   url     = {https://deepmind.com/research/publications/beta-VAE-Learning-Basic-Visual-Concepts-with-a-Constrained-Variational-Framework}
```
Comment (zoq) [inline src/mlpack/methods/ann/layer/alpha_dropout.hpp]: Hm, wrong line.

Comment (codeboy5) [reply src/mlpack/methods/ann/layer/alpha_dropout.hpp]: Sorry for the confusion.

**Comment by zoq** [comment]
Thanks again for adding the links.

**Pull request merged**